src/components/Pagination/styles.scss

```diff
@@ -0,0 +1,108 @@
+@import "../../styles/margins";
+@import "../../styles/colors";
+@import "../../styles/shadows";
+@import "../../styles/font-sizes";
+
+@mixin variants-common-styles($background-color, $border-color, $color) {
+    background-color: $background-color;
+    border: 1px solid $border-color;
+    color: $color;
+}
+
+.center-items {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.rainbow-pagination {
+    width: fit-content;
+}
+
+.rainbow-pagination_container {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+    width: fit-content;
+}
+
+.rainbow-pagination_button {
+    height: 2rem;
+    width: 2rem;
+    border: 1px $color-gray-2 solid;
+    margin-left: -1px;
+    background-color: white;
+
+    @extend .center-items;
+
+    a {
+        color: $color-gray-4;
+        line-height: unset;
+        width: 100%;
+        height: 100%;
+        text-decoration: none;
+        font-weight: bold;
+        font-size: $font-size-text-large;
+
+        @extend .center-items;
+    }
+}
+
+.rainbow-pagination_button--active {
+    background-color: $color-gray-2;
+
+    a {
+        color: $color-dark-1;
+    }
+}
+
+.rainbow-pagination_button--first {
```

Comment:
why you need add this classNames. You can use a pseudo-clase for example
the first element
```.rainbow-pagination_button:nth-of-type(1)```
the last element
```.rainbow-pagination_button:nth-last-of-type(1)```